java/ql/src/utils/testFooter.qlfrag

```diff
@@ -1,38 +0,0 @@
-
```

Comment:
Since this file is removed, shouldn't we remove this line then?
https://github.com/github/codeql/blob/57d44b8a40d7ac68b03330736b271efa1fc2c41d/java/ql/src/utils/GenerateFlowTestCase.py#L215

Reply:
Without question, yes. Not sure why I didn't think of that